app/stylesheets/boilerplate/view-mode.scss

```diff
@@ -1,12 +1,38 @@
 @import '../config/colours';
 @import '../config/list';
+@import '../config/dimensions';
+@import '../config/type';
 
-.view-mode {
-  @include list-inline();
+.view-mode-wrapper {
+  display: table-cell;
+  vertical-align: top;
+  width: 100%;
+}
+
+ul.view-mode {
```

Comment:
So, if the `.view-mode-wrapper` class just becomes `.view-mode` (see comment above), lets nest this `ul` inside it, and use the selector `& > ul`.